src/index.ts

```diff
@@ -107,6 +106,22 @@ function swc(options: PluginOptions = {}): RollupPlugin {
           ? {}
           : getOptions(this, dirname(id), options.tsconfig);
 
+      let enaleExperimentalDecorators = false;
```

Comment:
-> `enableExperimentalDecorators` ?

Reply:
I noticed it. But i don't know what is better :(

Reply:
I mean, it's a typo ((